types.go

```diff
@@ -6,4 +6,5 @@ type Params struct {
 	UploadURL string   `json:"upload_url"`
 	APIKey    string   `json:"api_key"`
 	Files     []string `json:"files"`
+	Checksums []string `json:"checksums"`
```

Comment:
Sounds good.  I'll add it in and change `checksums` to `checksum`, I like `checksum` better too, but I was worried it might be miss leading for the user.